packages/lesswrong/components/posts/PostsPage/BestOfLessWrong/PostWithArtGrid.tsx

```diff
@@ -85,7 +85,7 @@ export const PostWithArtGrid = ({post, images, defaultExpanded = false}: {post:
   const { create: createSplashArtCoordinateMutation } = useCreate({
     collectionName: 'SplashArtCoordinates',
     fragmentName: 'SplashArtCoordinatesEdit'
-  });   
+  });
```

Comment:
?

Reply:
I mean it looks like there used to be random spaces there, and it seems better than not to remove them.